src/lumigo_opentelemetry/resources/detectors.py

```diff
@@ -102,13 +103,18 @@ def detect(self) -> "Resource":
 
 def get_resource(attributes: dict) -> "Resource":
     return get_aggregated_resources(
-        detectors=[
-            OTELResourceDetector(),
-            EnvVarsDetector(),
-            ProcessResourceDetector(),
-            LumigoDistroDetector(),
-            AwsEcsResourceDetector(),
-            LumigoAwsEcsResourceDetector(),
-        ],
+        detectors=_get_detector_list(),
         initial_resource=Resource.create(attributes=attributes),
     )
+
+
+def _get_detector_list():
+    logging.getLogger("opentelemetry.sdk.extension.aws.resource.ecs").setLevel(logging.CRITICAL)
```

Comment:
It feels to me like this code should be in the beginning of the distro init, next to the other code we have for loggers

Reply:
I think here it's more specific for the detector and not like in the init() where it's more general. Also thinking about the future - if we'll need to suppress other detectors logs then the right place to do it is here

Reply:
Yes but the question is going to be “why don’t I see XYZ logs”, and people will naturally first look at https://github.com/lumigo-io/opentelemetry-python-distro/blob/925285d811ef6a3fe5def4b7b8503dae7241378d/src/lumigo_opentelemetry/__init__.py#L10. That is where this code should live with a comment about it.